technical-steering-committee/CHARTER.md

```diff
@@ -0,0 +1,126 @@
+# Technical Charter
+
+Technical Charter (the "Charter") for OpenSearch Project a Series of LF Projects, LLC
+
+_Adopted September 13, 2024_
+
+This Charter sets forth the responsibilities and procedures for technical contribution to, and oversight of, the OpenSearch Project open-source project, which has been established as OpenSearch Project a Series of LF Projects, LLC (the "Project"). LF Projects, LLC ("LF Projects") is a Delaware series limited liability company. All contributors (including committers, maintainers, and other technical positions) and other participants in the Project (collectively, "Collaborators") must comply with the terms of this Charter. 
+
+1. **Mission and Scope of the Project**
+
+   a. The mission of the Project is to develop an open-source search and analytics suite used by developers to perform various tasks, including but not limited to interactive log analytics, real-time application monitoring and website search, vector search for RAG and Generative A.I. use-cases, etc.
+
+   b. The scope of the Project includes collaborative development under the Project License (as defined herein) supporting the mission, including documentation, testing, integration and the creation of other artifacts that aid the development, deployment, operation or adoption of the open-source project.
+
+2. **Technical Steering Committee**
+
+   a. The Technical Steering Committee (the "TSC") will be responsible for all technical oversight of the open-source Project. 
+
+   b. Startup Period.
+
+      1. The initial voting members ("Initial Members") of the TSC at the inception of the Project (the "Start Date") are those individuals identified as voting members in the "STEERING.md" file within the Project’s .github repository. Initial Members will consist of Initial Members serving one-year terms and Initial Members serving two-year terms, as indicated in the STEERING.md file.
```

Comment:
I believe we set on TSC to be one year term?

Reply:
We always need some continuity, so it's 2 years to create overlap. At start LC members are 1 year so we can re-elect half after year 1, and the rest 2 years so we can re-elect the other half and then it's business as usual.